File: ASP_net/CWC_DOTNET_Final/CWC.Domain/Entities/Vehicule.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CWC.Domain.Entities
{
   public class Vehicule: Product
    {
        public int WheelsNbr { get; set; }
        public int Capacity { get; set; }
        public String VehiculeType { get; set; }  
        public float weightveh { get; set; }
    }
}

File: JavaEE+JSF/CWC_JEE_FINAL/cwc-bis-final/cwc-bis-ejb/src/main/java/tn/esprit/bi/cwc/persistance/Vehicule.java
package tn.esprit.bi.cwc.persistance;

import java.io.Serializable;

import javax.persistence.Entity;

/**
 * Entity implementation class for Entity: Vehicule
 *
 */
@Entity

public class Vehicule extends Product implements Serializable {

	private static final long serialVersionUID = 1L;
	private Integer WheelsNbr;
	private Integer Capacity;
	private String VehiculeType;
	private float weightveh;

	public Integer getWheelsNbr() {
		return WheelsNbr;
	}

	public void setWheelsNbr(Integer wheelsNbr) {
		WheelsNbr = wheelsNbr;
	}

	public Integer getCapacity() {
		return Capacity;
	}

	public void setCapacity(Integer capacity) {
		Capacity = capacity;
	}

	public String getVehiculeType() {
		return VehiculeType;
	}

	public void setVehiculeType(String vehiculeType) {
		VehiculeType = vehiculeType;
	}

	public float getWeightveh() {
		return weightveh;
	}

	public void setWeightveh(float weightveh) {
		this.weightveh = weightveh;
	}

	public Vehicule() {
		super();
	}

}
